src/components/modal/profile/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useContext } from 'react';
+import Backdrop from '@mui/material/Backdrop';
+import Modal from '@mui/material/Modal';
+import Fade from '@mui/material/Fade';
+import Typography from '@mui/material/Typography';
+import Avatar from '@mui/material/Avatar';
+import { LoadingButton } from '@mui/lab';
+import avatar from '../../../static/images/john-doe.png';
+import { Box } from '@mui/material';
+import { InteractiveComponentsContext } from 'contexts/interactiveComponentsContext';
+import IconButton from '@mui/material/IconButton';
+import CloseIcon from '@mui/icons-material/Close';
+import { useTheme } from '@mui/material/styles';
+
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  width: 400,
+  bgcolor: 'background.paper',
+  border: '2px solid #000',
+  boxShadow: 24,
+  p: 4,
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  overflow: 'hidden',
+};
+
+export const ProfileModal = () => {
+  const { isProfileModalOpened, closeProfileModal } = useContext(InteractiveComponentsContext);
```

Comment:
this function doesn't even exist in the InteractiveComponentsContext

Reply:
This change wasn't pushed at that time. This should be addressed by now.